test/e2e_env/universal/trafficroute/traffic_route.go

```diff
@@ -155,47 +154,6 @@ conf:
 		)
 	})
 
-	It("should route split traffic between the versions with 20/80 ratio", func() {
```

Comment:
There are other places where we check these kind of setup so this test was redundant